xyzspaces/spaces.py

```diff
@@ -1326,3 +1326,39 @@ def add_features_geopandas(
                 features_size=features_size,
                 chunk_size=chunk_size,
             )
+
+    def clone(self, space_id: str = None, limit: int = 1000):
+        """
+        Copy current space data into a new space.
```

Comment:
Be more specific if the target space will be newly created or can be an existing one.